File: app/src/main/java/com/example/test/CardEntity.java
package com.example.test;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "historyBin2")
public class CardEntity {

    @PrimaryKey
    @NonNull
    private String bin;
    private String scheme;
    private String type;
    private String bankName;
    private String timestamp;

    public CardEntity(@NonNull String bin, String scheme, String type, String bankName, String timestamp) {
        this.bin = bin;
        this.scheme = scheme;
        this.type = type;
        this.bankName = bankName;
        this.timestamp = timestamp;
    }

    public String getBin() {
        return bin;
    }

    public String getScheme() {
        return scheme;
    }

    public String getType() {
        return type;
    }

    public String getBankName() {
        return bankName;
    }

    public String getTimestamp() {
        return timestamp;
    }

    @Override
    public String toString() {
        return "CardEntity{" +
                "bin='" + bin + '\'' +
                ", scheme='" + scheme + '\'' +
                ", type='" + type + '\'' +
                ", bankName='" + bankName + '\'' +
                ", timestamp=" + timestamp +
                '}';
    }
}

File: app/src/main/java/com/example/test/data/model/CardEntity.kt
package com.example.test.data.model

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "historyBin2")
class CardEntity (
    @PrimaryKey val bin: String,
    val scheme: String,
    val type: String,
    val bankName: String,
    val timestamp: String
    ) {
    override fun toString(): String {
        return "CardEntity(bin='$bin', scheme='$scheme', type='$type', bankName='$bankName', timestamp='$timestamp')"
    }
}